dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -316,15 +313,11 @@ public static UnresolvableArtifactProblem createUnresolvableArtifactProblem(
    * just follows the given dependency tree starting with firstNode.
    *
    * @param firstNode node to start traversal
-   * @param graphTraversalOption option to recursively resolve the dependency to build complete
-   *     dependency tree, with or without dependencies of provided scope
    */
-  private DependencyGraphResult levelOrder(
-      DependencyNode firstNode, GraphTraversalOption graphTraversalOption) {
+  private DependencyGraph levelOrder(DependencyNode firstNode) {
```

Comment:
No need to wrap DependencyGraph with DependencyGraphResult. Because this function does not encounter `UnresolvedArtifactProblem` any more.